available_images.md

```diff
@@ -87,20 +87,20 @@ You can pin your version by adding the version tag to your URL as follows:
 E3 Framework Containers (EC2, ECS, and EKS support only)
 ============================
 
-| Framework         |Job Type	|Horovod Options|CPU/GPU 	|Python Version Options	|Example URL																						|
-|-------------------|-----------|---------------|-----------|-----------------------|---------------------------------------------------------------------------------------------------|
-|PyTorch 1.10.0   |training	|No			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.10.0-cpu-py38-ubuntu20.04-e3		|
-|PyTorch 1.10.0   |training	|Yes			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.10.0-gpu-py38-cu113-ubuntu20.04-e3	|
-|PyTorch 1.10.0   |inference	|No			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-inference:1.10.0-cpu-py38-ubuntu20.04-e3		|
-|PyTorch 1.10.0   |inference	|No			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-inference:1.10.0-gpu-py38-cu113-ubuntu20.04-e3	|
-|TensorFlow 2.8.0   |training	|Yes			|CPU 		| 3.9 (py39)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.8.0-cpu-py39-ubuntu20.04-e3		|
-|TensorFlow 2.8.0   |training	|Yes			|GPU 		| 3.9 (py39)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.8.0-gpu-py39-cu112-ubuntu20.04-e3	|
-|TensorFlow 2.8.0   |inference	|Yes			|CPU 		| 3.9 (py39)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.8.0-cpu-py39-ubuntu20.04-e3		|
-|TensorFlow 2.8.0   |inference	|Yes			|GPU 		| 3.9 (py39)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.8.0-gpu-py39-cu112-ubuntu20.04-e3	|
-|MXNet 1.9.0   |training	|Yes			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.9.0-cpu-py38-ubuntu20.04-e3		|
-|MXNet 1.9.0  |training	|Yes			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.9.0-gpu-py38-cu112-ubuntu20.04-e3	|
-|MXNet 1.9.0   |inference	|No			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-inference:1.9.0-cpu-py38-ubuntu20.04-e3        |
-|MXNet 1.9.0   |inference	|No			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-inference:1.9.0-gpu-py38-cu112-ubuntu20.04-e3	|
+| Framework        |Job Type	|Horovod Options|CPU/GPU 	|Python Version Options	| Example URL																						                                                                      |
+|------------------|-----------|---------------|-----------|-----------------------|--------------------------------------------------------------------------------------------------------|
+| PyTorch 1.10.2   |training	|No			|CPU 		| 3.8 (py38)			| 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.10.2-cpu-py38-ubuntu20.04-e3		         |
```

Comment:
Thanks for making the change for PT1.10.2 and rectifying the spaces. Shall we add PT1.10.0 to `Prior E3 Framework Container Versions` section beneath?

Reply:
Also, lets close this PR https://github.com/aws/deep-learning-containers/pull/1747/files once we get this merged. Thanks!!

Reply:
Thank you for opening the PR for PT 1.10.2 before. Didn't get a chance to review it before.

Reply:
Got it, thanks!!